sui/coin/sources/coin_witness.move

```diff
@@ -1,13 +1,13 @@
-module coin::coin_witness {
+module coin::coin {
```

Comment:
Should we rename this file `coin.move` to match this module name change?

Reply:
good idea